package.json

```diff
@@ -348,6 +348,7 @@
     "core-js": "^3.9.1",
     "date-fns": "^2.15.0",
     "fuse.js": "^6.4.6",
+    "jest-junit": "^13.0.0",
```

Comment:
This should be in devDependencies

Reply:
And needs a security review

Reply:
Wow. I called
```
npm install --save --dev jest-junit
```
instead of
```
npm install --save-dev jest-junit
```
and npm, the great tool that it is, just ran with it. No warning, no error.

In fact, I see
```
npm install --hurr-durr jest-junit
```
also runs successfully. Smh.